src/main/webapp/jsp/studentHome.jsp

```diff
@@ -7,7 +7,7 @@
 <c:set var="jsIncludes">
   <script type="text/javascript" src="/js/studentHome.js"></script>
 </c:set>
-<ts:studentPage pageTitle="TEAMMATES - Student" bodyTitle="Student Home" jsIncludes="${jsIncludes}">
+<ts:studentPage pageTitle="Home" bodyTitle="Student Home" jsIncludes="${jsIncludes}">
```

Comment:
Just highlighting that if we drop the `pageTitle` attribute, the title would also be `Student Home`.